tests/test_parse_samplesheet.py

```diff
@@ -12,7 +12,7 @@ def test_samplesheet():
 
 def test_runinfo():
     path = os.path.dirname(os.path.abspath(__file__))
-    k=classes.RunInfoParser(os.path.join(path, '../test_data/150424_ST-E00214_0031_BH2WY7CCXX/RunInfo.xml'))
+    k=classes.RunInfoParser(os.path.join(path, '../test_data/150424_ST-E00214_0031_BH2WY7CCXX/RunInfo.xml')
```

Comment:
@vezzi : `Travis` is failing because of this. The `RunInfoParser` misses the closing ')' causes syntax error.